cleanlab/multiannotator.py

```diff
@@ -0,0 +1,436 @@
+import numpy as np
+import pandas as pd
+from typing import List, Union, Tuple
+from cleanlab.rank import get_label_quality_scores
+from cleanlab.dataset import overall_label_health_score, _get_worst_class
+import warnings
+
+# TODO: unsure where to place this function? should it go to cleanlab.rank or stay here
+def _get_label_quality_scores_with_NA(
+    labels: pd.Series,
+    pred_probs: np.array,
+    kwargs: dict = {},
+) -> pd.Series:
+    label_quality_scores_subset = get_label_quality_scores(
+        labels=labels[pd.notna(labels)],
+        pred_probs=pred_probs[pd.notna(labels)],
+        **kwargs,
+    )
+    label_quality_scores = pd.Series(np.nan, index=np.arange(len(labels)))
+    label_quality_scores[pd.notna(labels)] = label_quality_scores_subset
+    return label_quality_scores
+
+
+def get_consensus_labels(
+    labels_multiannotator: pd.DataFrame,
+    pred_probs: np.array,
+    method: str,
+) -> np.array:
+    """Returns consensus labels aggregated from multiple annotators, determined by method specified.
+
+    Parameters
```

Comment:
To reduce docstring duplication, let's try to refer to the documentation of `get_label_quality_scores_multiannotator()` and `compute_multiannotator_stats()`

as much as possible in these helper methods. Those two are the main functions that should have the most complete-looking documentation.